CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ x.x.x Release notes (yyyy-MM-dd)
   non-nil value when passed in an invalid URL.
 * `SyncSession.Progress.fractionTransferred` now returns 1 if there are no
   transferrable bytes.
+* Change the way sync progress notifications are dispatched to fix broken
```

Comment:
This doesn't announce that they're no longer dispatched on the same thread on which they were registered, which is technically a breaking change, but we've always allowed ourselves to make breaking changes to already broken implementations, so that's fine.

However, I'd reword this:

> * Fix sync progress notifications registered on background threads by always dispatching on a dedicated background queue.